src/translate.spec.js

```diff
@@ -60,4 +63,46 @@ describe('translate enhancer', () => {
             expect(TranslatedComponent.displayName).toBe(translatedDefaultName);
         });
     });
+
+    it('should not re-render component on every non-related dispatch call', async () => {
+        let pChanged = false;
+        let dispatched = true;
```

Comment:
Sure, but I have some doubt about the utility of this boolean for this test, what do you think ?